### DISCUSSION TITLE
ESP32 +  Spfd5408 controller shield gives white screen

### DISCUSSION THREAD

**Discussion by sp3p3x** [Bodmer/TFT_eSPI#1989] [Compatible displays and setup files]
I have a nodemcu esp-32s and a 2.4" tft lcd shield. 
link to board - [https://quartzcomponents.com/products/esp32-development-board-wifi-bluetooth](url)
link to display - [https://quartzcomponents.com/products/2-4-inch-tft-lcd-display-module-for-arduino-uno](url)
this is my User_Setup.h file:

```
#define ESP32_PARALLEL
#define ILI9341_DRIVER

// ESP32 pins used for the parallel interface TFT
#define TFT_CS   27  
#define TFT_DC   14 
#define TFT_RST  26 
#define TFT_WR   12
#define TFT_RD   13

#define TFT_D0   16 
#define TFT_D1   4 
#define TFT_D2   23
#define TFT_D3   22
#define TFT_D4   21
#define TFT_D5   19
#define TFT_D6   18
#define TFT_D7   17

#define LOAD_GLCD   // Font 1. Original Adafruit 8 pixel font needs ~1820 bytes in FLASH
#define LOAD_FONT2  // Font 2. Small 16 pixel high font, needs ~3534 bytes in FLASH, 96 characters
#define LOAD_FONT4  // Font 4. Medium 26 pixel high font, needs ~5848 bytes in FLASH, 96 characters
#define LOAD_FONT6  // Font 6. Large 48 pixel font, needs ~2666 bytes in FLASH, only characters 1234567890:-.apm
#define LOAD_FONT7  // Font 7. 7 segment 48 pixel font, needs ~2438 bytes in FLASH, only characters 1234567890:.
#define LOAD_FONT8  // Font 8. Large 75 pixel font needs ~3256 bytes in FLASH, only characters 1234567890:-.
#define LOAD_GFXFF  // FreeFonts. Include access to the 48 Adafruit_GFX free fonts FF1 to FF48 and custom fonts

#define SMOOTH_FONT
```

i have been troubleshooting for the past two days but none of the fixes work.. i tried making different connections, checked the connections for continuity, different versions of tft_espi library, different spi frequencies, changing 3v3 to 5v etc. but none of these worked.. i tried running TFT_ReadWrite_Test with the above setup file and this is the output:
![image](https://user-images.githubusercontent.com/83198303/187046878-57ec27c3-76bb-4fdb-a411-156d034be075.png)
my connections:
[connections.zip](https://github.com/Bodmer/TFT_eSPI/files/9438639/connections.zip)

i went through all of the ILI9341 issues on the issues page and tried almost all of the troubleshooting steps.. none works..
i highly doubt if there is any compatibility problems with either the board or display since i've seen people say they changed their board/display and everything started working... 

any help is highly appreciated.. thanks in advance :)

**Comment by Bodmer** [comment]
The vendors page states Spfd5408 controller, so it is not an ILI9341 display.

The Spfd5408 controller is not supported by TFR_eSPI.
